wifiphisher/pywifiphisher.py

```diff
@@ -297,6 +298,10 @@ def try_change_mac(self, iface_name, mac_address=None):
             print("[{0}!{1}] {2}").format(R, W, err)
 
     def start(self):
+        dependency_result = check_dependencies()
+        if not dependency_result.status:
+            logger.error("{} is needed but not installed."
+                         "Please install and run again".format(dependency_result.name))
```

Comment:
If a strong dependency is missing, we should not proceed.

Reply:
Maybe this is also a good place to check for `roguehostapd`. We are currently doing the check in `accesspoint.py` (see <a href="https://github.com/wifiphisher/wifiphisher/blob/master/wifiphisher/common/accesspoint.py#L209">here</a>) but I believe we should move it here.

Reply:
Yes I forgot to add it. On that note we have two options to exit. First is to use `system.exit` and the second is `stop()` function. Which one would better?

Reply:
Maybe we can just use system.exit since we haven't allocate resource yet. Also maybe we can output in stdout for the dependencies missing since users may not provide --logging option.

Reply:
Correct.